packages/gatsby-source-wordpress/package.json

```diff
@@ -1,51 +1,93 @@
 {
   "name": "gatsby-source-wordpress",
-  "description": "Gatsby source plugin for building websites using the WordPress CMS as a data source.",
-  "version": "3.11.0",
-  "author": "Sebastien Fichot <fichot.sebastien@gmail.com>",
+  "description": "Source data from WordPress in an efficient and scalable way.",
+  "author": "Tyler Barnes <tylerdbarnes@gmail.com>",
+  "version": "4.0.0-next.0",
```

Comment:
The only change from the original PR here: replaced `4.1.0-next.0` with `4.0.0-next.0` as we'll publish 4.0.0 from this release branch.

Reply:
Also had to update dependency versions (missed that at first): https://github.com/gatsbyjs/gatsby/pull/29330/commits/988413c391d0f64ee66d6d828d984a76557aef3a

Reply:
Also looks like `yarn.lock` wasn't updated in the original PR.